system_metrics_collector/src/topic_statistics_collector/subscriber_topic_statistics.hpp

```diff
@@ -80,13 +80,23 @@ class SubscriberTopicStatisticsNode : public system_metrics_collector::MetricsMe
 
     const auto collect_topic_desc = BuildTopicParameterDescriptor(
       "The topic to subscribe to, for calculating topic statistics.");
-    collect_topic_name_ = declare_parameter(
+      // collect_topic_names_ = { "/hello_test", "/test_hello_topic" };
+    collect_topic_names_ = declare_parameter(
       constants::kCollectStatsTopicNameParam,
-      std::string() /* setting default to empty, since this is a required parameter */,
+      std::vector<std::string>() /* setting default to empty, since this is a required parameter */,
       collect_topic_desc);
-    ValidateStringParam(
-      constants::kCollectStatsTopicNameParam,
-      collect_topic_name_);
+
+    if (collect_topic_names_.empty()) {
+      CheckEmptyStringParam(constants::kCollectStatsTopicNameParam, std::string());
```

Comment:
This is going to always fail.
Is the intent to fail when `collect_topic_names_` is empty?
In that case L89-91 should be:
```c++
rcpputils::check_true(!collect_topic_names_.empty(), <meaningful message here>);
```

Reply:
Resolving this comment in favor of https://github.com/ros-tooling/system_metrics_collector/pull/143#discussion_r417460026